### PR TITLE
Update ACL code for pre 10.13.4 to use a const authorization tag.

### PR DESCRIPTION
**Pull request by rohitnarula7176**

### CODE REVIEW

File: IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m [hunk 1]
```diff
@@ -1126,8 +1127,19 @@ - (id)accessCreateWithChangeACL:(NSArray<id> *)trustedApplications error:(NSErro
         return nil;
     }
     
+    CFStringRef authorizationTag;
+    
+    if (@available(macOS 10.13.4, *))
+    {
+        authorizationTag = kSecACLAuthorizationChangeACL;
+    } else
```
Comment (oldalton) [inline IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m]: nit: new line before else

File: IdentityCore/src/MSIDConstants.m [hunk 2]
```diff
@@ -61,3 +61,5 @@
 
 NSString *const MSID_DEFAULT_AAD_AUTHORITY       = @"https://login.microsoftonline.com/common";
 NSString *const MSID_DEFAULT_MSA_TENANTID        = @"9188040d-6c67-4c5b-b112-36a304b66dad";
+
+NSString *const MSID_ACL_OWNER_AUTHORIZATION_TAG = @"ACLAuthorizationChangeACL";
```
Comment (oldalton) [inline IdentityCore/src/MSIDConstants.m]: nit: I'd keep this inside MSIDMacKeychainTokenCache

File: IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m (hunk 1 above)
Comment (rohitnarula7176) [reply IdentityCore/src/cache/mac/MSIDMacKeychainTokenCache.m]: Addressed.

File: IdentityCore/src/MSIDConstants.m (hunk 2 above)
Comment (rohitnarula7176) [reply IdentityCore/src/MSIDConstants.m]: Addressed.

**Pull request merged**